src/Relude/Applicative.hs

```diff
@@ -30,6 +32,17 @@ import Control.Applicative (Alternative (..), Applicative (..), Const (..), ZipL
 pass :: Applicative f => f ()
 pass = pure ()
 
+{- | For chaining applicative operations in forward applications using '(&)'
+
+>>> Just (+ 1) & appliedTo (Just 2)
+Just 3
+>>> Nothing & appliedTo (Just 2)
+Nothing
+-}
+appliedTo :: Applicative f => f a -> f (a -> b) -> f b
+appliedTo x f = f <*> x
+
```

Comment:
Let's add `{-# INLINE #-}` pragma for this small function to have more guarantees on performance.
```suggestion
{-# INLINE appliedTo #-}
```